ONSdigital/census-rm-acceptance-tests-2021
# Run acceptance tests in CI Pipeline

# Motivation and Context
We want to start running the acceptance tests in our concourse pipelines. Currently the tests need to be run in kubernetes so the tests need to have automated docker builds. This PR also adds the concourse task so it will live in this repo. The task runs the tests with the `kubectl run` command. It creates a single use pod which is deleted when finished and runs the acceptance tests while attached so that concourse gets the test logs and exit code.

# What has changed
- Add travis yml for automated image builds
- Add concourse task to run the tests in kubernetes
- Add kubernetes env file

# How to test?
Check the rm team pipelines, this task has been used to successfully run the tests against my project.

# Links
https://github.com/ONSdigital/census-rm-deploy/pull/7
https://trello.com/c/o5zK0iaP/533-run-acceptance-tests-in-ci-pipeline
